packages/Search/src/details/DTK_DetailsUtils.hpp

```diff
@@ -147,13 +148,39 @@ void iota( Kokkos::View<T, P...> const &v,
         std::is_same<ValueType, typename Kokkos::ViewTraits<
                                     T, P...>::non_const_value_type>::value,
         "iota requires a View with non-const value type" );
-    auto const n = v.span();
+    auto const n = v.extent( 0 );
     Kokkos::parallel_for(
         "iota", Kokkos::RangePolicy<ExecutionSpace>( 0, n ),
         KOKKOS_LAMBDA( int i ) { v( i ) = value + (ValueType)i; } );
     Kokkos::fence();
 }
 
+/** \brief Returns the smallest and the greatest element in the view
+ *
+ *  \param[in] v Input view
+ *
+ *  Returns a pair with the smallest value in the view as the first
+ *  element and the greatest as the second.
+ */
+template <typename ViewType>
+std::pair<typename ViewType::non_const_value_type,
```

Comment:
Maybe use `Kokkos::pair`?

Reply:
What is the benefit to use `Kokkos::pair` if the pair is returned on the host?

Reply:
Not sure, just wanted for you to take a look at it and tell us :)